<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockSetFace</class>
 <widget class="QWidget" name="ClockSetFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>320</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AR PL UKai CN</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>300</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="ldt_s">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>52</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldt_tm">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>100</y>
      <width>53</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldt_h">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>100</y>
      <width>52</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>93</y>
      <width>10</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>93</y>
      <width>10</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>93</y>
      <width>52</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>52</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-color:rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>日期:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldt_y">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>17</y>
      <width>58</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2013</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldt_d">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>17</y>
      <width>58</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>213</x>
      <y>10</y>
      <width>12</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldt_m">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>17</y>
      <width>57</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>10</y>
      <width>12</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai CN</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="tbtn_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tbtn_cancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai CN</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
